core/sail/api/src/main/java/org/eclipse/rdf4j/sail/features/ConcurrentReadsSailConnection.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Eclipse RDF4J contributors.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Distribution License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/org/documents/edl-v10.php.
+ ******************************************************************************/
+
+package org.eclipse.rdf4j.sail.features;
+
+/**
+ * And interface used to signal that the SailConnection supports concurrent reads on the same connection.
+ */
+public interface ConcurrentReadsSailConnection {
```

Comment:
I'd name this more generally, e.g. `ThreadSafetyAware`. That gives us (some) license in future to add more related signal methods, if needed.

Reply:
Sure.